db/schema.sql

```diff
@@ -136,7 +136,7 @@ CREATE TABLE users_purchases(
 	status_url VARCHAR(255) NULL,
 	qrcode_url VARCHAR(255) NOT NULL,
 	payment_received BOOLEAN NOT NULL DEFAULT FALSE,
-	permission VARCHAR(255) NOT NULL,
+	permission VARCHAR(255) NOT NULL DEFAULT "community",
```

Comment:
I like this that we defaults to community

Reply:
I agree! I thought that we should have community be default and then users could specify further privacy if they prefer that. This way when we are inserting into the users_purchases table we don't have to worry about if the user did not select a permission value.